api/api.go

```diff
@@ -98,6 +100,11 @@ type httpOkProofs struct {
 	RegisteredValidators   []uint64 `json:"registered_validators"`
 }
 
+type httpOkRelayersState struct {
+	RegisteredRelayers   string `json:"registered_relayers"`
```

Comment:
Add discussed, can we also have a `Registered` field that stores `true` or `false`?